### PR TITLE
Fix violations of and reenable rubocop Layout/EmptyLineAfterMagicComment.

### PR DESCRIPTION
**Pull request by asherkach**

### CODE REVIEW

**Comment by asherkach** [comment]
@breville for review (as DOTD).

File: dashboard/app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # coding: utf-8
```
Comment (breville) [inline dashboard/app/helpers/application_helper.rb:1]: Given that the default encoding in Ruby 2.0 and higher is `utf-8`, do we actually need these lines?

Comment (asherkach) [reply dashboard/app/helpers/application_helper.rb:1]: Probably not. But I'm going to call that a job for another PR.

Comment (breville) [reply dashboard/app/helpers/application_helper.rb:1]: Ah, is this now superseded by https://github.com/code-dot-org/code-dot-org/pull/23954?

Comment (asherkach) [reply dashboard/app/helpers/application_helper.rb:1]: Yes, it is (the fixes).

After that PR gets merged, I'll reenable the rubocop.

**Pull request closed without merging**